mvrx/src/test/kotlin/com/airbnb/mvrx/ViewSubscriberTest.kt

```diff
@@ -15,20 +15,20 @@ class ViewSubscriberFragment : BaseMvRxFragment() {
     private val viewModel: ViewSubscriberViewModel by fragmentViewModel()
 
     var subscribeCallCount = 0
-    var selectSubscribeCalled = 0
+    var selectSubscribeCalled = -1
```

Comment:
This needs to be -1 to properly assert what we want, otherwise the first call to the selectSusbcribe lambda sets 0 to 0